src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import {useState, useEffect} from 'react'
+import Header from './components/Header'
+import LocationSelector from "./components/LocationSelector/index.jsx";
+import ResultsDisplay from "./components/ResultsDisplay/index.jsx";
+import Footer from "./components/Footer/index.jsx";
+import {BrowserRouter} from "react-router-dom";
+
+function App() {
+
+    const [stations, setStations] = useState([])
+    const [selected, setSelected] = useState(null)
```

Comment:
If this is the start of the journey, it would be clearer to call this something like `fromSelected`